test_start_stop_ioc_through_proc_control.py

```diff
@@ -138,27 +138,39 @@ def test_WHEN_start_iocs_THEN_iocs_started_WHEN_stop_iocs_THEN_iocs_stopped(self
         self.assertGreater(len(iocs), 100)
         # Check there's at least one known ioc in the list
         self.assertTrue(any(item in iocs for item in ["SIMPLE", "AMINT2L_01", "EUROTHRM_01", "INSTETC_01"]))
```

Comment:
I'm happy to move those out if you think its absolutely nessecary, but I think these are meant to be a "don't bother testing the iocs if there not sensible" rather than an actual test. possibly be better just to change this to not be an assert but to raise an exception so that its clear this is an error not a test failure?

Reply:
I agree with you, I think changing the level of exception to error makes more sense here.